src/main/plugins/org.dita.html5/xsl/syntax-braces.xsl

```diff
@@ -85,62 +94,86 @@ See the accompanying LICENSE file for applicable license.
   
   <!-- set up group containers (similar to same area management as for syntaxdiagram, synblk,  and fragment) -->
   
-  <xsl:template match="syntaxdiagram/*[contains(@class,' pr-d/groupcomp ')]|syntaxdiagram/*[contains(@class,' pr-d/groupseq ')]|syntaxdiagram/*[contains(@class,' pr-d/groupchoice ')]" mode="process-syntaxdiagram">
+  <xsl:template match="syntaxdiagram/*[contains(@class,' pr-d/groupcomp ') or contains(@class,' syntaxdiagram-d/groupcomp ')] |
+                       syntaxdiagram/*[contains(@class,' pr-d/groupseq ') or contains(@class,' syntaxdiagram-d/groupseq ')] |
+                       syntaxdiagram/*[contains(@class,' pr-d/groupchoice ') or contains(@class,' syntaxdiagram-d/groupchoice ')]"
+                mode="process-syntaxdiagram">
     <xsl:call-template name="dogroup"/>
   </xsl:template>
   
   
   <!-- handle group titles (by skipping over them for now -->
   
-  <xsl:template match="*[contains(@class,' pr-d/groupcomp ')]/*[contains(@class,' topic/title ')]|*[contains(@class,' pr-d/groupseq ')]/*[contains(@class,' topic/title ')]|*[contains(@class,' pr-d/groupseq ')]/*[contains(@class,' topic/title ')]" mode="process-syntaxdiagram"/>  <!-- Consume title -->
```

Comment:
This was supposed to match title for all 3 groups, but doubled up groupseq -- `groupcomp/title` + `groupseq/title` + `groupseq/title`. Fixed the last to `groupchoice`